### PR TITLE
CC-26541 Adjusted IGs according changes related to code api conventions refactoring

### PR DESCRIPTION
**Pull request by ievgen-malykhin**
## PR Description

Ticket: https://spryker.atlassian.net/browse/CC-30355

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202400.0/install-the-product-offer-service-points-feature.md [hunk 1]
```diff
@@ -84,37 +84,37 @@ Make sure that propel entities have been generated successfully by checking thei
...
-| Filter                                 | class | created | src/Generated/Shared/Transfer/FilterTransfer                                 |
+| TRANSFER                                  | TYPE  | EVENT   | PATH                                                                            |
+|-------------------------------------------|-------|---------|---------------------------------------------------------------------------------|
+| ServicePoint                              | class | created | src/Generated/Shared/Transfer/ServicePointTransfer                              |
+| ServicePointCollection                    | class | created | src/Generated/Shared/Transfer/ServicePointCollectionTransfer                    |
+| ServicePointCriteria                      | class | created | src/Generated/Shared/Transfer/ServicePointCriteriaTransfer                      |
+| ServicePointConditions                    | class | created | src/Generated/Shared/Transfer/ServicePointConditionsTransfer                    |
+| ServicePointAddressesBackendApiAttributes | class | created | src/Generated/Shared/Transfer/ServicePointAddressesBackendApiAttributesTransfer |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-product-offer-service-points-feature.md:93]: Strange, as I see this DTO part of the Service Points feature, please remove it from the connector feature.

File: _includes/pbc/all/install-features/202400.0/install-the-warehouse-user-management-feature.md [hunk 2]
```diff
@@ -65,8 +65,9 @@ Make sure that the following changes have been triggered in transfer objects:
...
 | WarehouseUserAssignmentCollectionResponse       | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionResponse       |
 | WarehouseUserAssignmentCollectionDeleteCriteria | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionDeleteCriteria |
-| ApiWarehouseUserAssignmentsAttributes           | class    | created    | src/Generated/Shared/Transfer/ApiWarehouseUserAssignmentsAttributesTransfer   |
-| ApiWarehousesAttributes                         | class    | created    | src/Generated/Shared/Transfer/ApiWarehousesAttributes                         |
+| WarehouseUserAssignmentsBackendApiAttributes    | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentsBackendApiAttributes    |
+| WarehousesBackendApiAttributes                  | class    | created    | src/Generated/Shared/Transfer/WarehousesBackendApiAttributes                  |
+| UsersBackendApiAttributes                       | class    | created    | src/Generated/Shared/Transfer/UsersBackendApiAttributes                       |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-warehouse-user-management-feature.md:70]: The same question for `UsersBackendApiAttributes`, it's part of `Spryker Core Back Office` feature. Is it somewhere defined in the warehouse user connector?

File: _includes/pbc/all/install-features/202400.0/install-the-product-offer-service-points-feature.md (hunk 1 above)
Comment (ievgen-malykhin) [reply _includes/pbc/all/install-features/202400.0/install-the-product-offer-service-points-feature.md:93]: Removed:
- ServicePointCollection
- ServicePointCriteria
- ServicePointConditions
- ServicePointAddressesBackendApiAttributes

File: _includes/pbc/all/install-features/202400.0/install-the-warehouse-user-management-feature.md (hunk 2 above)
Comment (ievgen-malykhin) [reply _includes/pbc/all/install-features/202400.0/install-the-warehouse-user-management-feature.md:70]: Didn't found in related modules. So UsersBackendApiAttributes transfer was removed.

**Review by dmiseev — DISMISSED** [review]

File: _includes/pbc/all/install-features/202400.0/install-the-packaging-units-feature.md [hunk 3]
```diff
@@ -0,0 +1,1138 @@
...
+  * `amount_measurement_unit_precision`
+  * `amount_measurement_unit_conversion`
+  * `amount_base_measurement_unit_name`
+
+Go to the Zed UI Sales overview, check the order, and verify the following:
+- The correct sales unit is displayed.
+- The correct amount is displayed per sales order item.
+
+Make sure the following:
+* Abstract products that have packaging units available don’t have `add_to_cart_sku` field at Elasticsearch document.
```
Comment (lenadoc) [inline _includes/pbc/all/install-features/202400.0/install-the-packaging-units-feature.md:963]: "Elasticsearch document" seems wrong here.

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: _includes/pbc/all/install-features/202400.0/install-the-packaging-units-feature.md (hunk 3 above)
Comment (ievgen-malykhin) [reply _includes/pbc/all/install-features/202400.0/install-the-packaging-units-feature.md:963]: Could you specify please what exactly is wrong?
We use ElasticSearch as a search engine.
I checked the case when the product has no packaging units: there is an `add_to_cart_sku` field in the elastic search document. But the product which has packaging units has no such field. That's what is mentioned here.

**Pull request merged**